migration_toolkit_for_containers/release_notes/mtc-release-notes.adoc

```diff
@@ -17,6 +17,7 @@ You can migrate from xref:../../migrating_from_ocp_3_to_4/about-migrating-from-3
 
```

Comment:
Line 14 says it is possible to migrate from OCP 3 to OCP 4 , This should be removed from 1.8 release notes.

Reply:
Thanks removed